src/Resources/views/CRUD/base_edit.html.twig

```diff
@@ -15,6 +15,8 @@ file that was distributed with this source code.
     {# NEXT_MAJOR: remove default filter #}
     {% if objectId|default(admin.id(object)) is not null %}
         {{ 'title_edit'|trans({'%name%': admin.toString(object)|truncate(15) }, 'SonataAdminBundle') }}
+        {# NEXT_MAJOR: Remove previous line and uncomment the following one #}
+        {# {{ 'title_edit'|trans({'%name%': admin.subjectAsString()|truncate(15) }, 'SonataAdminBundle') }} #}
```

Comment:
It would be great to have a twig extension just for this. That way, Twig does not know or care what object has the `subjectAsString()` method.

Reply:
Looks like that's already what you did above :thinking: why not here?

Reply:
IMO, the extension should be mainly focused on the admin context: an admin which have a subject.
I guess it could (optionally) work with standalone objects.

Regarding this question, I don't get what you mean:

>Looks like that's already what you did above thinking why not here?

Reply:
I meant that on the previous code snippet, you are using a new twig function instead of calling a method of the admin object

Reply:
Oh, I see.
Because where I've used `subject_to_string` Twig filter, the admin has no subject.